ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/KafkaStreamsStateCount.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.confluent.ksql.util.KsqlException;
+
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.kafka.streams.KafkaStreams;
+
+/**
+ * Used to keep track of a the state of KafkaStreams application
+ * across multiple servers. Used in {@link RunningQuery}.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class KafkaStreamsStateCount {
```

Comment:
I'm a little confused by this class.  Why does it have all this code to convert to and from a string representation in the form `<state>: <count>, <state>: <count>`?  Why not just return this as a JSON Object?  The Jackson does all the heavy lifting for you!

```json
 {
     "CREATED": 1,
     "RUNNING": 2
}
```

This can be achieved with:

```java
@JsonIgnoreProperties(ignoreUnknown = true)
public class KafkaStreamsStateCount {

  private final Map<State, Integer> states;

  public KafkaStreamsStateCount() {
    this.states = new HashMap<>();
  }

  @SuppressWarnings("unused") // Invoked by reflection
  @JsonCreator
  public KafkaStreamsStateCount(final Map<State, Integer> states) {
    this.states = new HashMap<>(states);
  }

  public void updateStateCount(final String state, final int change) {
    updateStateCount(KafkaStreams.State.valueOf(state), change);
  }

  public void updateStateCount(final KafkaStreams.State state, final int change) {
    states.compute(state, (key, existing) ->
        existing == null
            ? change
            : existing + change
    );
  }

  @JsonValue
  public Map<KafkaStreams.State, Integer> getState() {
    return states;
  }

  @Override
  public boolean equals(final Object o) {
    if (this == o) {
      return true;
    }

    if (o == null || getClass() != o.getClass()) {
      return false;
    }

    final KafkaStreamsStateCount that = (KafkaStreamsStateCount) o;
    return Objects.equals(states, that.states);
  }

  @Override
  public int hashCode() {
    return Objects.hash(states);
  }
}
```

Reply:
it's returned as a json object now